lib/livereload.js

```diff
@@ -122,9 +123,11 @@ function StartServer() {
                 // Usually, this err msg ends up being null in this case.
                 var error = new Error('No External URLs available. Make sure your computer is connected to a network.' + (err ? err.msg : ''));
                 deferred.reject(error);
+                return;
             }
 
             deferred.resolve(server_url);
+	    return;
```

Comment:
Yeah, I don't wanna force beautify to run automatically on every save coz when I do it ends up messing pre-existing code formatting. :)